### ISSUE TITLE
Add economic calculations to oemof

### ISSUE THREAD

**Issue by simnh** [oemof/oemof#431]
I think it would be great to just have some economic calculations based on a standard approach (e.g. NREL or VDI-standard). Maybe, if somebody has some money, we could just pay a HiWi for that...? 

(we already have the annuity, yeah!), but LCOE etc. would be nice to, and I guess if you start thinking about it, there is much more...

**Comment by henhuy** [comment]
I just made calculations for LCOE, OPEX, Investment, Fuel Costs.
They depend on the new param_results from PR #416 
Once this is approved, I will start a PR for my implementations

**Comment by henhuy** [comment]
Hey @simnh - I just added PR #433 to cover this issue.
Would be cool if you could review too!

**Comment by uvchik** [comment]
> Would be cool if you could review too!

If you want somebody to be a reviewer you can select him within the PR (top right). The person will be noticed.

**Comment by p-snft** [comment]
As #543 just covers improved annuity calculation and I do not see e.g. LCOE coming in the near future, I will postpone this feature request.
